### ISSUE TITLE
Players should be able to choose their own faction

### ISSUE THREAD

**Issue by smcameron** [smcameron/space-nerds-in-space#230]
When a new player ship is created, a faction is assigned randomly.  The players are not given any clear indication of what their own faction is.  It is *possible* to discover (but not at all obvious how). To find out what your own faction is, you have to get on COMMS and type in:

```
/computer select *your-ship-name*
```

Then go on SCIENCE, and you'll see your own faction.Human player's ship is assigned a random faction, and it is difficult to discover

The player's faction determines whet It would be better if players could easily know their own faction (or even choose their own faction -- perhaps even in game, dynamically)her various enemy ships of other factions are friendly or hostile or neutral.

Players should be able to easily find out their own faction, and also choose their faction at ship creation time, and possibly change their faction during the game, perhaps at any time (though this could potentially lead to mischief in multi-crew, so not sure about that) or perhaps at designated times during game play (e.g. maybe have to dock at a starbase to change faction?)

**Comment by smcameron** [comment]
Commit f7b28f9d409746c2120c5d977bdf16f22d57452a allows players to choose their own faction at ship creation time.

However, this doesn't help in the case of ships which already exist.

Perhaps it should be possible to change factions in-game.  Perhaps at a starbase, perhaps in exchange for some amount of in game currency.

**Comment by MCMic** [comment]
I expect a lot of mission scripts would be broken if the player change faction in the middle of it.

**Comment by smcameron** [comment]
Good point about the mission scripts.  Will have to think about that some more.  Perhaps mission scripts could inhibit faction-changing via a tweakable variable?  Or maybe something else will come to mind.  As it stands, factions are assigned once at ship creation (before now, randomly, and now you may choose, or if you don't choose, randomly).  There is no subsequent chance to change factions -- the faction is saved in the "database" (if that isn't too strong a word for how things are saved) and persists between sessions.  Part of what brought this notion of changing factions to mind was a (spoiler alert) discussion of Empty Epsilon: http://bridgesim.net/discussion/455/empty-epsilon-scenario-escape-question-spoilers#latest

Apparently EE simulates some kind of [IFF](https://en.wikipedia.org/wiki/Identification_friend_or_foe) system which you can mess with.  (i.e. masquerade as another faction, I guess.)  Seems like there are some possibilities there which would be nice to, er, make possible.

**Comment by smcameron** [comment]
Added a Lua script to allow setting the faction of any ship from the demon screen: f488c88757904cc83f064cc0f77b61a96ac79443
